pkg/cmd/convert.go

```diff
@@ -62,20 +69,44 @@ func GetValidatedObject(v *viper.Viper, cmd *cobra.Command, out, outerr io.Write
 	}
 
 	var ocObjects []*object.OpenCompose
+	var data []byte
+	var err error
 	for _, file := range files {
```

Comment:
@surajssd I would say it is really the last think we should be discussing at this point and it is **not a subject of this PR**!